meta-raspberrypi: Whitelisting components and plugins which are

Khan3033 opened the pull request:
having ristricted license "commercial".And enabling vc4graphics.

gstreamer1.0-plugins-bad : Enabling debug utils and packaging getreamer
plugins.

Signed-off-by: Riyaz <Riyaz.l@ltts.com>

<!--
Please make sure you've read and understood our contributing guidelines.

For additional information on the contribution guidelines:
https://wiki.yoctoproject.org/wiki/Contribution_Guidelines#General_Information

If this PR fixes an issue, make sure your description includes "fixes #xxxx".

If this PR connects to an issue, make sure your description includes "connected to #xxxx".

Please provide the following information:
-->

**- What I did**

**- How I did it**

Commit:
meta-raspberrypi: Whitelisting components and plugins which are
having ristricted license "commercial".And enabling vc4graphics.

gstreamer1.0-plugins-bad : Enabling debug utils and packaging getreamer
plugins.

Signed-off-by: Riyaz <Riyaz.l@ltts.com>

Commit:
Enabling vc4graphics machine feature.

gstreamer1.0-plugins-bad : Enabling debug utils and packaging getreamer
plugins.

Signed-off-by: Riyaz <Riyaz.l@ltts.com>

kraj commented:
Please post these changes against meta-raspberrypi upstream I think setting commercial license is not right change here we have to let the developer select that

psivasubramanian commented:
@Khan3033 Please open this PR against https://github.com/agherzan/meta-raspberrypi (http://git.yoctoproject.org/cgit/cgit.cgi/meta-raspberrypi).

Commit:
Enabling vc4graphics machine feature.

gstreamer1.0-plugins-bad : Enabling debug utils and packaging getreamer
plugins.

Commit:
Merge branch 'master' of https://github.com/Khan3033/meta-raspberrypi

Commit:
Enabling vc4graphics machine feature.

gstreamer1.0-plugins-bad : Enabling debug utils and packaging getreamer
plugins.
